libs/langchain/langchain/document_loaders/readthedocs.py

```diff
@@ -67,33 +70,14 @@ def load(self) -> List[Document]:
             metadata = {"source": str(p)}
             docs.append(Document(page_content=text, metadata=metadata))
         return docs
-
+    
     def _clean_data(self, data: str) -> str:
         from bs4 import BeautifulSoup
-
+        
         soup = BeautifulSoup(data, **self.bs_kwargs)
-
-        # default tags
-        html_tags = [
-            ("div", {"role": "main"}),
-            ("main", {"id": "main-content"}),
-        ]
-
-        if self.custom_html_tag is not None:
-            html_tags.append(self.custom_html_tag)
-
-        text = None
-
-        # reversed order. check the custom one first
-        for tag, attrs in html_tags[::-1]:
-            text = soup.find(tag, attrs)
-            # if found, break
-            if text is not None:
-                break
-
-        if text is not None:
-            text = text.get_text()
-        else:
-            text = ""
-        # trim empty lines
-        return "\n".join([t for t in text.split("\n") if t])
+           
+        text = soup.get_text().strip()
```

Comment:
It's not clear why this code would result in better results. In order to get these changes accepted, you'd need to add unit tests showing different scenarios where the parse results in better data.